en/docs/references/about-this-release.md

```diff
@@ -98,6 +98,38 @@ The following new features and enhancements (introduced in WSO2 IS 6.0.0) are in
 
     [Learn more]({{base_path}}/deploy/configure-recaptcha/)
 
+-  **Tenant qualified URLs and tenanted sessions**
+
+    From IS 7.0.0 onwards, tenant qualified URLs will be enabled by default, which consistently qualifies every URL/ endpoint of WSO2 Identity Server with the tenant in a path parameter. This improves flexibility for tenant-wise sharding and branding compared to previous releases.
+
+    Additionally from IS 7.0.0 onwards, tenanted sessions will also be enabled allowing sessions and caches to be created in a tenanted space.
+
+    !!! info
+        It is strongly recommended to use WSO2 Identity Server with these two configs being enabled. However, if you need to disable any of these configs, follow the below steps.
+        
+        1. Add the following constraint to the "IDN_OAUTH_CONSUMER_APPS" table by executing the following sql command against the identity database (IDENTITY_DB).
+
+            ```
+            ALTER TABLE IDN_OAUTH_CONSUMER_APPS ADD CONSTRAINT UNIQUE_CONSUMER_KEY_CONSTRAINT UNIQUE (CONSUMER_KEY);
+            ```
+
+        2. Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory and add the following configuration.
+        
+            ```
+            [tenant_context]
+            enable_tenant_qualified_urls = false
+            enable_tenanted_sessions = false
```

Comment:
We need to emphasize that we cannot switch back these configs

Reply:
Removed since no need to document these configs